### ISSUE TITLE
Incorporate CLI and YAML editor colors into light and dark themes

### ISSUE THREAD

**Issue by roraback** [gravitational/teleport#25413]
YAML color details TBD.
CLI uses named colors from the xterm package at https://xtermjs.org/docs/api/terminal/interfaces/itheme/

Defaults for the CLI colors appear to be defined in [ThemeService.ts within that package's GitHub repo](https://github.com/xtermjs/xterm.js/blob/1b13127f0be68b7e4b1a6c71717c74f503ff783d/src/browser/services/ThemeService.ts#L23):

```
const DEFAULT_FOREGROUND = css.toColor('#ffffff');      // confirmed in CLI
const DEFAULT_BACKGROUND = css.toColor('#000000');      // confirmed in CLI
const DEFAULT_CURSOR = css.toColor('#ffffff');
const DEFAULT_CURSOR_ACCENT = css.toColor('#000000');
const DEFAULT_SELECTION = {
  css: 'rgba(255, 255, 255, 0.3)',
  rgba: 0xFFFFFF4D
};
black: '#2e3436',
red: '#cc0000',
green: '#4e9a06',
yellow: '#c4a000',
blue: '#3465a4',
magenta: '#75507b',
cyan: '#06989a',
white: '#d3d7cf',
brightBlack: '#555753',
brightRed: '#ef2929',       // confirmed in CLI
brightGreen: '#8ae234',     // confirmed in CLI
brightYellow: '#fce94f',
brightBlue: '#729fcf',      // confirmed in CLI
brightMagenta: '#ad7fa8',   // confirmed in CLI
brightCyan: '#34e2e2',
brightWhite: '#eeeeec'
```

**Comment by roraback** [comment]
Helpful link from @zmb3 with popular terminal themes: https://iterm2colorschemes.com/

**Comment by roraback** [comment]
Here is a [first draft of CLI theming](https://www.figma.com/file/Gpjs9vjhzUKF1GDbeG9JGE/Application-Design-System?type=design&node-id=7405%3A35959&t=GiVo2KBfGUNf5bad-1) with some assumptions and open questions listed

**Comment by roraback** [comment]
Proposed dark theme CSS for the Ace YAML editor. Ideally light theme will be extrapolated from the dark theme choices.

Some comments are from the theme builder; all caps are my commentary and references to our design theme colors.

```
.teleport-dark {
    background-color: #0C143D; /* Overall background color - DEPTH 3 */
    color: #FFFFFF; /* Overall foreground color - FOREGROUND DEFAULT */
}

.teleport-dark .ace_gutter { /* Editor gutter contains of line numbers and annotation marks. */
    background: #222C59; /* Background color of the editor gutter. - DEPTH 2*/
    color: rgba(255,255,255,.54); /* Color of line numbers - FOREGROUND MUTED */
}

.teleport-dark .ace_gutter-cell {
    padding-right: 15px; /* Increase the gutter padding so the carets aren't quite so close to the edge */
}

.teleport-dark .ace_print-margin { /* Print margin line - I HAVE NO IDEA WHAT THIS IS */
    width: 1px;
    background: #0C143D; /* DEPTH 3 */
}

.teleport-dark .ace_cursor {
    color: #FFFFFF; /* Color of the editor cursor - DEFAULT FOREGROUND */
}

.teleport-dark .ace_marker-layer .ace_selection {
    background: rgba(255,255,255,.18); /* Color of the editor selection - SPOT BG 3 */
}

.teleport-dark.ace_multiselect .ace_selection.ace_start { /* Ace multi selections - DEPTH 3 */
    box-shadow: 0 0 3px 0px #0C143D; /* DEPTH 3 */
}

.teleport-dark .ace_marker-layer .ace_step { /* Debugger line -  */
    background: rgba(255, 171, 0, 0.5); /* NOT TESTED IN THE WILD; NOT RELEVANT FOR YAML */
}

.teleport-dark .ace_marker-layer .ace_active-line {
    background: rgba(255, 255, 255, 0.13); /* Background color of active editor's line - SPOT 2 */
}

.teleport-dark .ace_gutter-active-line {
    background-color: #344179; /* Background color of gutter active line - DEPTH 1 */
}

.teleport-dark .ace_marker-layer .ace_selected-word { /* Style of selected words. Try it by double clicking on any word in Result tab*/
    box-shadow: 0px 0px 0px 1px #fff;
    border-radius: 2px;

}

/* .teleport-dark .ace_fold { /* Fold widget - BUT IS IT? These styles were provided in the widget and I see them in other themes, but everything I see in the inspector is .ace_fold_widget rather than .ace_fold; this may be a legacy style reference? */
    /* background-color: #0f0; /* Background of fold widget 
    /* border-color: white; /* Border color of fold widget */
/*}*/

.teleport-dark .ace_fold_widget { /* The ACTUAL fold widget. We just want to make it a hair wider without breaking the layout */
    width: 13px;
    margin-right: -14px;
}


/* Start: different token styles - GROUPINGS TAKEN FROM Tomorrow Night Blue theme */

.teleport-dark .ace_invisible {
  color: #4A5688; /* DEPTH 0 */
}

.teleport-dark .ace_keyword,
.teleport-dark .ace_meta,
.teleport-dark .ace_storage,
.teleport-dark .ace_storage.ace_type,
.teleport-dark .ace_support.ace_type {
  color: #B9A6FF; /* DARK/DATA/PURPLE/TERTIARY */
}

.teleport-dark .ace_keyword.ace_operator {
  color: #74EEFF; /* DARK/DATA/CYAN/TERTIARY */
}

.teleport-dark .ace_constant.ace_character,
.teleport-dark .ace_constant.ace_language,
.teleport-dark .ace_constant.ace_numeric,
.teleport-dark .ace_keyword.ace_other.ace_unit,
.teleport-dark .ace_support.ace_constant,
.teleport-dark .ace_variable.ace_parameter {
  color: #FF948D; /* DARK/DATA/ABBEY/TERTIARY */
}

.teleport-dark .ace_invalid {
  color: #FFFFFF; /* FOREGROUND DEFAULT */
  background-color: #DB3F34; /* DARK/DATA/ABBEY/SECONDARY */
}

.teleport-dark .ace_invalid.ace_deprecated {
  color: #FFFFFF; /* FOREGROUND DEFAULT */
  background-color: #B9A6FF; /* DARK/DATA/PURPLE/TERTIARY */
}

.teleport-dark .ace_fold {
  background-color: #7BCDFF; /* DARK/DATA/PICTON/TERTIARY */
  border-color: #FFFFFF; /* FOREGROUND DEFAULT */
}

.teleport-dark .ace_entity.ace_name.ace_function,
.teleport-dark .ace_support.ace_function,
.teleport-dark .ace_variable {
  color: #7BCDFF /* DARK/DATA/PICTON/TERTIARY */
}

.teleport-dark .ace_support.ace_class,
.teleport-dark .ace_support.ace_type {
  color: #FFD98C; /* DARK/DATA/SUNFLOWER/TERTIARY */
}

.teleport-dark .ace_heading,
.teleport-dark .ace_markup.ace_heading,
.teleport-dark .ace_string {
  color: #2EFFD5; /* DARK/DATA/CARIBBEAN/TERTIARY */
}

.teleport-dark .ace_entity.ace_name.ace_tag,
.teleport-dark .ace_entity.ace_other.ace_attribute-name,
.teleport-dark .ace_meta.ace_tag,
.teleport-dark .ace_string.ace_regexp,
.teleport-dark .ace_variable {
  color: #B9A6FF; /* DARK/DATA/PURPLE/TERTIARY */
}

.teleport-dark .ace_comment {
  color: rgba(255, 255, 255, 0.54); /* FOREGROUND MUTED */
}

/* End: different token styles */

.teleport-dark .ace_indent-guide { /* Indent guide style */
    background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAACCAYAAACZgbYnAAAAEklEQVQImWNgYGBgYHB3d/8PAAOIAdULw8qMAAAAAElFTkSuQmCC) right repeat-y;
}

.teleport-dark .ace_indent-guide-active { /* Active indent guide style */
    background: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAACAQMAAACjTyRkAAAABlBMVEUAAADCwsK76u2xAAAAAXRSTlMAQObYZgAAAAxJREFUCNdjYGBoAAAAhACBGFbxzQAAAABJRU5ErkJggg==") right repeat-y;
}
```

**Comment by roraback** [comment]
@zmb3 - a big thank you for sending me the link to the Ace editor! If you want to see the theme in action, [follow this link](https://mkslanc.github.io/ace-playground/?value=Ly9zYW1wbGVzL2NyZWF0aW5nIG5ldyB0aGVtZQoKdmFyIHZhbHVlID0gYGtpbmQ6IHJvbGUKbWV0YWRhdGE6CiAgZGVzY3JpcHRpb246IEFjY2VzcyBjbHVzdGVyIHJlc291cmNlcwogIG5hbWU6IGFjY2VzcwpzcGVjOgogIGFsbG93OgogICAgYXBwX2xhYmVsczoKICAgICAgJyonOiAnKicKICAgIGF3c19yb2xlX2FybnM6CiAgICAtICd7e2ludGVybmFsLmF3c19yb2xlX2FybnN9fScKICAgIGF6dXJlX2lkZW50aXRpZXM6CiAgICAtICd7e2ludGVybmFsLmF6dXJlX2lkZW50aXRpZXN9fScKICAgIGRiX2xhYmVsczoKICAgICAgJyonOiAnKicKICAgIGRiX25hbWVzOgogICAgLSAne3tpbnRlcm5hbC5kYl9uYW1lc319JwogICAgLSBwb3N0Z3JlcwogICAgZGJfc2VydmljZV9sYWJlbHM6CiAgICAgICcqJzogJyonCiAgICBkYl91c2VyczoKICAgIC0gJ3t7aW50ZXJuYWwuZGJfdXNlcnN9fScKICAgIC0gYWxpY2UKICAgICMgLSBFeGFtcGxlVGVsZXBvcnREeW5hbW9EQlJvbGUKICAgIGdjcF9zZXJ2aWNlX2FjY291bnRzOgogICAgLSAne3tpbnRlcm5hbC5nY3Bfc2VydmljZV9hY2NvdW50c319JwogICAga3ViZXJuZXRlc19ncm91cHM6CiAgICAtICd7e2ludGVybmFsLmt1YmVybmV0ZXNfZ3JvdXBzfX0nCiAgb3B0aW9uczoKICAgIGNlcnRfZm9ybWF0OiBzdGFuZGFyZAogICAgZm9yd2FyZF9hZ2VudDogdHJ1ZQogICAgaWRwOgogICAgICBzYW1sOgogICAgICAgIGVuYWJsZWQ6IHRydWUKICAgIG1heF9zZXNzaW9uX3R0bDogMzBoMG0wcwogICAgcGluX3NvdXJjZV9pcDogZmFsc2UKICAgIHBvcnRfZm9yd2FyZGluZzogdHJ1ZQogICAgcmVjb3JkX3Nlc3Npb246CiAgICAgIGRlc2t0b3A6IHRydWUKICAgIHNzaF9maWxlX2NvcHk6IHRydWUKdmVyc2lvbjogdjYKYDsKCnZhciBlZGl0b3IgPSBhY2UuZWRpdCgiZXhhbXBsZSIsIHsKICAgIG1vZGU6ICJhY2UvbW9kZS95YW1sIiwKICAgIHZhbHVlOiB2YWx1ZQp9KTsKCi8vQ2xvc2Ugb25lIGxldmVsIHRvIHNlZSBmb2xkIHdpZGdldApzZXRUaW1lb3V0KCgpID0%2BIGVkaXRvci5zZXNzaW9uLmZvbGRUb0xldmVsKDEwKSwgMTAwKTsKCi8vIEVkaXRvcidzIHRoZW1lIGNvdWxkIGJlIGN1c3RvbWlzZWQgYnkgcHJvdmlkaW5nIHlvdXIgb3duIGNzcyBzdHlsZXMKZWRpdG9yLnNldFRoZW1lKHtjc3NDbGFzczogInRlbGVwb3J0LWRhcmsiLCBpc0Rhcms6IHRydWV9KVwwLnRlbGVwb3J0LWRhcmsgewogICAgYmFja2dyb3VuZC1jb2xvcjogIzBDMTQzRDsgLyogT3ZlcmFsbCBiYWNrZ3JvdW5kIGNvbG9yIC0gREVQVEggMyAqLwogICAgY29sb3I6ICNGRkZGRkY7IC8qIE92ZXJhbGwgZm9yZWdyb3VuZCBjb2xvciAtIEZPUkVHUk9VTkQgREVGQVVMVCAqLwp9CgoudGVsZXBvcnQtZGFyayAuYWNlX2d1dHRlciB7IC8qIEVkaXRvciBndXR0ZXIgY29udGFpbnMgb2YgbGluZSBudW1iZXJzIGFuZCBhbm5vdGF0aW9uIG1hcmtzLiAqLwogICAgYmFja2dyb3VuZDogIzIyMkM1OTsgLyogQmFja2dyb3VuZCBjb2xvciBvZiB0aGUgZWRpdG9yIGd1dHRlci4gLSBERVBUSCAyKi8KICAgIGNvbG9yOiByZ2JhKDI1NSwyNTUsMjU1LC41NCk7IC8qIENvbG9yIG9mIGxpbmUgbnVtYmVycyAtIEZPUkVHUk9VTkQgTVVURUQgKi8KfQoKLnRlbGVwb3J0LWRhcmsgLmFjZV9ndXR0ZXItY2VsbCB7CiAgICBwYWRkaW5nLXJpZ2h0OiAxNXB4OyAvKiBJbmNyZWFzZSB0aGUgZ3V0dGVyIHBhZGRpbmcgc28gdGhlIGNhcmV0cyBhcmVuJ3QgcXVpdGUgc28gY2xvc2UgdG8gdGhlIGVkZ2UgKi8KfQoKLnRlbGVwb3J0LWRhcmsgLmFjZV9wcmludC1tYXJnaW4geyAvKiBQcmludCBtYXJnaW4gbGluZSAtIEkgSEFWRSBOTyBJREVBIFdIQVQgVEhJUyBJUyAqLwogICAgd2lkdGg6IDFweDsKICAgIGJhY2tncm91bmQ6ICMwQzE0M0Q7IC8qIERFUFRIIDMgKi8KfQoKLnRlbGVwb3J0LWRhcmsgLmFjZV9jdXJzb3IgewogICAgY29sb3I6ICNGRkZGRkY7IC8qIENvbG9yIG9mIHRoZSBlZGl0b3IgY3Vyc29yIC0gREVGQVVMVCBGT1JFR1JPVU5EICovCn0KCi50ZWxlcG9ydC1kYXJrIC5hY2VfbWFya2VyLWxheWVyIC5hY2Vfc2VsZWN0aW9uIHsKICAgIGJhY2tncm91bmQ6IHJnYmEoMjU1LDI1NSwyNTUsLjE4KTsgLyogQ29sb3Igb2YgdGhlIGVkaXRvciBzZWxlY3Rpb24gLSBTUE9UIEJHIDMgKi8KfQoKLnRlbGVwb3J0LWRhcmsuYWNlX211bHRpc2VsZWN0IC5hY2Vfc2VsZWN0aW9uLmFjZV9zdGFydCB7IC8qIEFjZSBtdWx0aSBzZWxlY3Rpb25zIC0gREVQVEggMyAqLwogICAgYm94LXNoYWRvdzogMCAwIDNweCAwcHggIzBDMTQzRDsgLyogREVQVEggMyAqLwp9CgoudGVsZXBvcnQtZGFyayAuYWNlX21hcmtlci1sYXllciAuYWNlX3N0ZXAgeyAvKiBEZWJ1Z2dlciBsaW5lIC0gICovCiAgICBiYWNrZ3JvdW5kOiByZ2JhKDI1NSwgMTcxLCAwLCAwLjUpOyAvKiBOT1QgVEVTVEVEIElOIFRIRSBXSUxEOyBOT1QgUkVMRVZBTlQgRk9SIFlBTUwgKi8KfQoKLnRlbGVwb3J0LWRhcmsgLmFjZV9tYXJrZXItbGF5ZXIgLmFjZV9hY3RpdmUtbGluZSB7CiAgICBiYWNrZ3JvdW5kOiByZ2JhKDI1NSwgMjU1LCAyNTUsIDAuMTMpOyAvKiBCYWNrZ3JvdW5kIGNvbG9yIG9mIGFjdGl2ZSBlZGl0b3IncyBsaW5lIC0gU1BPVCAyICovCn0KCi50ZWxlcG9ydC1kYXJrIC5hY2VfZ3V0dGVyLWFjdGl2ZS1saW5lIHsKICAgIGJhY2tncm91bmQtY29sb3I6ICMzNDQxNzk7IC8qIEJhY2tncm91bmQgY29sb3Igb2YgZ3V0dGVyIGFjdGl2ZSBsaW5lIC0gREVQVEggMSAqLwp9CgoudGVsZXBvcnQtZGFyayAuYWNlX21hcmtlci1sYXllciAuYWNlX3NlbGVjdGVkLXdvcmQgeyAvKiBTdHlsZSBvZiBzZWxlY3RlZCB3b3Jkcy4gVHJ5IGl0IGJ5IGRvdWJsZSBjbGlja2luZyBvbiBhbnkgd29yZCBpbiBSZXN1bHQgdGFiKi8KICAgIGJveC1zaGFkb3c6IDBweCAwcHggMHB4IDFweCAjZmZmOwogICAgYm9yZGVyLXJhZGl1czogMnB4OwoKfQoKLyogLnRlbGVwb3J0LWRhcmsgLmFjZV9mb2xkIHsgLyogRm9sZCB3aWRnZXQgLSBCVVQgSVMgSVQ%2FIEV2ZXJ5dGhpbmcgSSBzZWUgaW4gdGhlIGluc3BlY3RvciBpcyAuYWNlX2ZvbGRfd2lkZ2V0OyB0aGlzIG1heSBiZSBhIGxlZ2FjeSBzdHlsZSByZWZlcmVuY2UgKi8KICAgIC8qIGJhY2tncm91bmQtY29sb3I6ICMwZjA7IC8qIEJhY2tncm91bmQgb2YgZm9sZCB3aWRnZXQgCiAgICAvKiBib3JkZXItY29sb3I6IHdoaXRlOyAvKiBCb3JkZXIgY29sb3Igb2YgZm9sZCB3aWRnZXQgKi8KLyp9Ki8KCi50ZWxlcG9ydC1kYXJrIC5hY2VfZm9sZF93aWRnZXQgeyAvKiBUaGUgQUNUVUFMIGZvbGQgd2lkZ2V0LiBXZSBqdXN0IHdhbnQgdG8gbWFrZSBpdCBhIGhhaXIgd2lkZXIgd2l0aG91dCBicmVha2luZyB0aGUgbGF5b3V0ICovCiAgICB3aWR0aDogMTNweDsKICAgIG1hcmdpbi1yaWdodDogLTE0cHg7Cn0KCgovKiBTdGFydDogZGlmZmVyZW50IHRva2VuIHN0eWxlcyAtIEdST1VQSU5HUyBUQUtFTiBGUk9NIFRvbW9ycm93IE5pZ2h0IEJsdWUgdGhlbWUgKi8KCi50ZWxlcG9ydC1kYXJrIC5hY2VfaW52aXNpYmxlIHsKICBjb2xvcjogIzRBNTY4ODsgLyogREVQVEggMCAqLwp9CgoudGVsZXBvcnQtZGFyayAuYWNlX2tleXdvcmQsCi50ZWxlcG9ydC1kYXJrIC5hY2VfbWV0YSwKLnRlbGVwb3J0LWRhcmsgLmFjZV9zdG9yYWdlLAoudGVsZXBvcnQtZGFyayAuYWNlX3N0b3JhZ2UuYWNlX3R5cGUsCi50ZWxlcG9ydC1kYXJrIC5hY2Vfc3VwcG9ydC5hY2VfdHlwZSB7CiAgY29sb3I6ICNCOUE2RkY7IC8qIERBUksvREFUQS9QVVJQTEUvVEVSVElBUlkgKi8KfQoKLnRlbGVwb3J0LWRhcmsgLmFjZV9rZXl3b3JkLmFjZV9vcGVyYXRvciB7CiAgY29sb3I6ICM3NEVFRkY7IC8qIERBUksvREFUQS9DWUFOL1RFUlRJQVJZICovCn0KCi50ZWxlcG9ydC1kYXJrIC5hY2VfY29uc3RhbnQuYWNlX2NoYXJhY3RlciwKLnRlbGVwb3J0LWRhcmsgLmFjZV9jb25zdGFudC5hY2VfbGFuZ3VhZ2UsCi50ZWxlcG9ydC1kYXJrIC5hY2VfY29uc3RhbnQuYWNlX251bWVyaWMsCi50ZWxlcG9ydC1kYXJrIC5hY2Vfa2V5d29yZC5hY2Vfb3RoZXIuYWNlX3VuaXQsCi50ZWxlcG9ydC1kYXJrIC5hY2Vfc3VwcG9ydC5hY2VfY29uc3RhbnQsCi50ZWxlcG9ydC1kYXJrIC5hY2VfdmFyaWFibGUuYWNlX3BhcmFtZXRlciB7CiAgY29sb3I6ICNGRjk0OEQ7IC8qIERBUksvREFUQS9BQkJFWS9URVJUSUFSWSAqLwp9CgoudGVsZXBvcnQtZGFyayAuYWNlX2ludmFsaWQgewogIGNvbG9yOiAjRkZGRkZGOyAvKiBGT1JFR1JPVU5EIERFRkFVTFQgKi8KICBiYWNrZ3JvdW5kLWNvbG9yOiAjREIzRjM0OyAvKiBEQVJLL0RBVEEvQUJCRVkvU0VDT05EQVJZICovCn0KCi50ZWxlcG9ydC1kYXJrIC5hY2VfaW52YWxpZC5hY2VfZGVwcmVjYXRlZCB7CiAgY29sb3I6ICNGRkZGRkY7IC8qIEZPUkVHUk9VTkQgREVGQVVMVCAqLwogIGJhY2tncm91bmQtY29sb3I6ICNCOUE2RkY7IC8qIERBUksvREFUQS9QVVJQTEUvVEVSVElBUlkgKi8KfQoKLnRlbGVwb3J0LWRhcmsgLmFjZV9mb2xkIHsKICBiYWNrZ3JvdW5kLWNvbG9yOiAjN0JDREZGOyAvKiBEQVJLL0RBVEEvUElDVE9OL1RFUlRJQVJZICovCiAgYm9yZGVyLWNvbG9yOiAjRkZGRkZGOyAvKiBGT1JFR1JPVU5EIERFRkFVTFQgKi8KfQoKLnRlbGVwb3J0LWRhcmsgLmFjZV9lbnRpdHkuYWNlX25hbWUuYWNlX2Z1bmN0aW9uLAoudGVsZXBvcnQtZGFyayAuYWNlX3N1cHBvcnQuYWNlX2Z1bmN0aW9uLAoudGVsZXBvcnQtZGFyayAuYWNlX3ZhcmlhYmxlIHsKICBjb2xvcjogIzdCQ0RGRiAvKiBEQVJLL0RBVEEvUElDVE9OL1RFUlRJQVJZICovCn0KCi50ZWxlcG9ydC1kYXJrIC5hY2Vfc3VwcG9ydC5hY2VfY2xhc3MsCi50ZWxlcG9ydC1kYXJrIC5hY2Vfc3VwcG9ydC5hY2VfdHlwZSB7CiAgY29sb3I6ICNGRkQ5OEM7IC8qIERBUksvREFUQS9TVU5GTE9XRVIvVEVSVElBUlkgKi8KfQoKLnRlbGVwb3J0LWRhcmsgLmFjZV9oZWFkaW5nLAoudGVsZXBvcnQtZGFyayAuYWNlX21hcmt1cC5hY2VfaGVhZGluZywKLnRlbGVwb3J0LWRhcmsgLmFjZV9zdHJpbmcgewogIGNvbG9yOiAjMkVGRkQ1OyAvKiBEQVJLL0RBVEEvQ0FSSUJCRUFOL1RFUlRJQVJZICovCn0KCi50ZWxlcG9ydC1kYXJrIC5hY2VfZW50aXR5LmFjZV9uYW1lLmFjZV90YWcsCi50ZWxlcG9ydC1kYXJrIC5hY2VfZW50aXR5LmFjZV9vdGhlci5hY2VfYXR0cmlidXRlLW5hbWUsCi50ZWxlcG9ydC1kYXJrIC5hY2VfbWV0YS5hY2VfdGFnLAoudGVsZXBvcnQtZGFyayAuYWNlX3N0cmluZy5hY2VfcmVnZXhwLAoudGVsZXBvcnQtZGFyayAuYWNlX3ZhcmlhYmxlIHsKICBjb2xvcjogI0I5QTZGRjsgLyogREFSSy9EQVRBL1BVUlBMRS9URVJUSUFSWSAqLwp9CgoudGVsZXBvcnQtZGFyayAuYWNlX2NvbW1lbnQgewogIGNvbG9yOiByZ2JhKDI1NSwgMjU1LCAyNTUsIDAuNTQpOyAvKiBGT1JFR1JPVU5EIE1VVEVEICovCn0KCi8qIEVuZDogZGlmZmVyZW50IHRva2VuIHN0eWxlcyAqLwoKLnRlbGVwb3J0LWRhcmsgLmFjZV9pbmRlbnQtZ3VpZGUgeyAvKiBJbmRlbnQgZ3VpZGUgc3R5bGUgKi8KICAgIGJhY2tncm91bmQ6IHVybChkYXRhOmltYWdlL3BuZztiYXNlNjQsaVZCT1J3MEtHZ29BQUFBTlNVaEVVZ0FBQUFFQUFBQUNDQVlBQUFDWmdiWW5BQUFBRWtsRVFWUUltV05nWUdCZ1lIQjNkLzhQQUFPSUFkVUx3OHFNQUFBQUFFbEZUa1N1UW1DQykgcmlnaHQgcmVwZWF0LXk7Cn0KCi50ZWxlcG9ydC1kYXJrIC5hY2VfaW5kZW50LWd1aWRlLWFjdGl2ZSB7IC8qIEFjdGl2ZSBpbmRlbnQgZ3VpZGUgc3R5bGUgKi8KICAgIGJhY2tncm91bmQ6IHVybCgiZGF0YTppbWFnZS9wbmc7YmFzZTY0LGlWQk9SdzBLR2dvQUFBQU5TVWhFVWdBQUFBRUFBQUFDQVFNQUFBQ2pUeVJrQUFBQUJsQk1WRVVBQUFEQ3dzSzc2dTJ4QUFBQUFYUlNUbE1BUU9iWVpnQUFBQXhKUkVGVUNOZGpZR0JvQUFBQWhBQ0JHRmJ4elFBQUFBQkpSVTVFcmtKZ2dnPT0iKSByaWdodCByZXBlYXQteTsKfQpcMDxkaXYgaWQ9J2V4YW1wbGUnIHN0eWxlPSJoZWlnaHQ6IDEwMCUiPjwvZGl2Pg%3D%3D#creating-new-theme) and paste some YAML code in. (Note: the URL was too long when I generated a link with a full YAML role definition, so I truncated it significantly. If you want to see a full definition, you'll need to copy/paste one into the editor.)
